torch/__init__.py

```diff
@@ -74,10 +74,11 @@
 
 # See Note [Global dependencies]
 def _load_global_deps():
-    if platform.system() == 'Windows':
-        return
-
-    lib_name = 'libtorch_global_deps' + ('.dylib' if platform.system() == 'Darwin' else '.so')
+    ext_dict = {'Darwin': '.dylib', 'Windows': '.dll'}
+    cur_platform = platform.system()
+    lib_name_prefix = '' if cur_platform == 'Windows' else 'lib'
+    lib_name_suffix = ext_dict.get(cur_platform, '.so')
+    lib_name = lib_name_prefix + 'torch_global_deps' + lib_name_suffix
```

Comment:
How about something like that:
```suggestion
    lib_name = {
        'Darwin': 'libtorch_global_deps.dylib', 
        'Linux': 'libtorch_global_deps.so',
        'Windows': 'torch_global_deps.dll',
         }.get(platform.system(), 'UNKNOWN')
```